app/src/main/java/org/mozilla/reference/browser/push/FirebasePush.kt

```diff
@@ -4,6 +4,8 @@
 
 package org.mozilla.reference.browser.push
 
+import android.annotation.SuppressLint
 import mozilla.components.lib.push.firebase.AbstractFirebasePushService
 
+@SuppressLint("MissingFirebaseInstanceTokenRefresh")
 class FirebasePush : AbstractFirebasePushService()
```

Comment:
CC @jonalmeida - Is that something we should just suppress?

Reply:
Yes, it's a safe change. 

We haven't needed to suppress it before despite the warning. I'm guess the AGP upgrade is making the build fail with warningsAsErrors.

Reply:
Yes, it was.